.github/workflows/test.yml

```diff
@@ -14,11 +14,11 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
-        django-version: [2.2]
+        python-version: ['3.11', '3.12', '3.13']
```

Comment:
Please use this [matrix](https://github.com/django-oscar/django-oscar/blob/f0179c430ea1d80469f4fe88f6709cd34ee34501/.github/workflows/test.yml#L20)

Reply:
Done 👍